tests/kola/version/rhaos-pkgs-match-openshift

```diff
@@ -9,6 +9,6 @@ set -xeuo pipefail
 
 . $KOLA_EXT_DATA/commonlib.sh
 
-if [[ $(rpm -qa | grep rhaos | grep -v $OPENSHIFT_VERSION) ]]; then
+if rpm -qa | grep "rhaos" | grep -qv "${OPENSHIFT_VERSION}"; then
```

Comment:
The way this conditional is structured makes it easy to have false positives. E.g. if `rpm` is completely broken for whatever reason, the test will still succeed.

I would reflow it as e.g.:

```bash
rpm -qa | grep rhaos > rhaos-packages.txt
if grep -qv ... rhaos-packages.txt; then
  fatal ...
fi
```

Reply:
Thanks @jlebon - if the considered rpm is completely broken, is it not the case that it will still show up in the `rhaos-packages.txt` ?

Reply:
Regardless, I updated the code as per the suggestion.